inspire/dojson/institutions/schemas/insitutions-0.0.1.json

```diff
@@ -8,13 +8,53 @@
             "type": "string"
         },
         "address": {
-            "description": "Street etc, city with postal code + additions (native language) FIXME: isn't postal code redundant?",
-            "title": "Address",
-            "type": "string"
-        },
-        "city": {
-            "title": "City",
-            "type": "string"
+            "items": {
+                "properties": {
+                    "address": {
+                        "description": "Street etc, city with postal code + additions (native language) FIXME: isn't postal code redundant?",
+                        "title": "Address",
+                        "type": "string"
+                    },
+                    "city": {
+                        "title": "City",
+                        "type": "string"
+                    },
+                    "state_province": {
+                        "title": "State or Province",
+                        "type": "string"
+                    },
+                    "country": {
+                        "description": "FIXME can't we derive this from country_code?",
+                        "enum": [
+                            "does",
+                            "it",
+                            "exist?"
+                        ],
+                        "string": "Country name",
+                        "type": "string"
+                    },
+                    "postal_code": {
+                        "title": "Postal code",
+                        "type": "string"
+                    },
+                    "country_code": {
+                        "enum": [
+                            "let's",
+                            "take",
+                            "the",
+                            "enum",
+                            "from",
+                            "iso"
+                        ],
+                        "title": "Country code",
+                        "type": "string"
+                    },
+                },
+                "title": "Address",
+                "type": "object"
+            },
+            "title": "Addresses",
+            "type": "array",
```

Comment:
I thought of it this way to prepare for possibly multiple addresses, so that you can then have coherent address structures (associated address, city, state, country in the same structure). We can always change it back if it's not feasible though. :-)